apps/sms/views/conversation/js/conversation.js

```diff
@@ -879,19 +870,18 @@ var ConversationView = {
   /**
    * Checks if specified panel corresponds to the specified conversation id. It
    * can be true for either conversation, participants or report panels.
-   * @param {number} conversationId Id of the conversation.
+   * @param {number} id Id of the conversation.
    * @param {Object} panel Panel description object to compare against.
    * @returns {boolean}
    */
-  isConversationPanel:
-  function conv_isConversationPanel(conversationId, panel) {
+  isConversationPanel(id, panel) {
     if (!panel) {
       return false;
     }
 
-    return panel.panel === 'thread' && panel.args.id === conversationId ||
-      panel.panel === 'report-view' && panel.args.threadId === conversationId ||
-      panel.panel === 'group-view' && panel.args.id === conversationId;
+    return panel.panel === 'thread' && +panel.args.id === +id ||
+      panel.panel === 'report-view' && +panel.args.threadId === +id ||
+      panel.panel === 'group-view' && +panel.args.id === +id;
```

Comment:
cosmetic change because I added "+" to convert from String and one line was too long